### PR TITLE
Updating LIP-A2 config to match HW revs.

### PR DESCRIPTION
**Pull request by yusufm**

### CODE REVIEW

File: configs/lip-a2/microscope.j5 [hunk 1]
```diff
@@ -1,35 +1,79 @@
...
+                "$100=500.000", "$101=500.000", "$102=500.000",
 
                 //2023-02-24: suspect missing steps. Make less aggressive
                 //X - Max Rate (mm/min)
                 //Default
                 //"$110=1200.000", "$111=1200.000", "$112=1200.000",
-                "$110=600.000", "$111=600.000", "$112=600.000",
+                //"$110=600.000", "$111=600.000", "$112=600.000",
+                //8/5 adjustment
+                "$110=960.000", "$111=960.000", "$112=1920.000",
```
Comment (JohnDMcMaster) [inline configs/lip-a2/microscope.j5:47]: revert this back to no more than 960. This is extra velocity bump as a compromise on adding the gearbox

File: configs/lip-a2/microscope.j5 [hunk 2]
```diff
@@ -1,35 +1,79 @@
...
                 //"$120=30.000", "$121=30.000", "$122=30.000",
-                "$120=15.000", "$121=15.000", "$122=15.000",
+                //8/5 adjustment
+                "$120=24.000", "$121=24.000", "$122=24.000",
 
+                //max travel
+                //"$130=300.000", "$131=200.000", "$132=60.000",
+                //"$130=480.000", "$131=320.000", "$132=96.000",
+                //z axis gearbox + 0.9 motor
+                "$130=480.000", "$131=320.000", "$132=960.000",
```
Comment (JohnDMcMaster) [inline configs/lip-a2/microscope.j5:58]: Z is incorrect. Needs to be 96.0 w/o the gearbox

File: configs/lip-a2/microscope.j5 [hunk 3]
```diff
@@ -1,35 +1,79 @@
...
+                //$25: Homing seek, mm/min
+                "$24=40.0","$25=800.000",
+                //$26: Homing debounce, milliseconds
+                "$26=400",
+                //$27: Homing pull-off, mm
+                //x/y: 24 * 5/8 => 15 mm
+                //z: 24 / (8/5 * 10) => 1.5 mm
+                //z:  48 => 3.0 mm
+                //2023-10-12: had increased this to be within soft limits, but it makes it take a really long time to home
+                "$27=5.000",
```
Comment (JohnDMcMaster) [inline configs/lip-a2/microscope.j5:74]: this is probably sub-optimal w/o gearbox (homing will take much longer than it needs to), but should be fine for now

File: configs/lip-a2/microscope.j5 [hunk 4]
```diff
@@ -41,83 +85,114 @@
                 "G54",
                 */
                 //More proper homed scheme
-                "G10 L2 P1 X-297.000 Y-197.000 Z-3.000",
+                //"G10 L2 P1 X-297.000 Y-197.000 Z-3.000",
+                //8/5 adjustment
+                //"G10 L2 P1 X-475.200 Y-315.200 Z-4.800",
+                //gearbox + 0.9 motor
+                "G10 L2 P1 X-475.200 Y-315.200 Z-48.000",
```
Comment (JohnDMcMaster) [inline configs/lip-a2/microscope.j5:92]: Z should be -4.8 as it doesn't have the gearbox

File: configs/lip-a2/microscope.j5 [hunk 5]
```diff
@@ -41,83 +85,114 @@
...
+
+    //E3ISPM20000KPA
+    "imager": {
+        "source":"gst-toupcamsrc",
+        "width": 5440,
+        "height": 3648,
+        "source_properties": {
+            "esize": 0
+        },
+        "scalar": 0.5,
```
Comment (JohnDMcMaster) [inline configs/lip-a2/microscope.j5:133]: We should review how much usable resolution / sensor size. But I imagine this is a fine starting point

**Review by JohnDMcMaster — CHANGES_REQUESTED** [review]

File: configs/lip-a2/microscope.j5 (hunk 5 above)
Comment (yusufm) [reply configs/lip-a2/microscope.j5]: 👍🏼

File: configs/lip-a2/microscope.j5 (hunk 4 above)
Comment (yusufm) [reply configs/lip-a2/microscope.j5]: done

File: configs/lip-a2/microscope.j5 (hunk 3 above)
Comment (yusufm) [reply configs/lip-a2/microscope.j5:72]: 👍🏼

File: configs/lip-a2/microscope.j5 (hunk 2 above)
Comment (yusufm) [reply configs/lip-a2/microscope.j5]: done

File: configs/lip-a2/microscope.j5 (hunk 1 above)
Comment (yusufm) [reply configs/lip-a2/microscope.j5]: done

**Comment by JohnDMcMaster** [comment]
This is probably close enough now that let's merge it and fix up later as needed

**Pull request merged**